business_objects/record.py

```diff
@@ -88,6 +91,48 @@ def get_ids_of_manual_records_by_labeling_task(
     )
 
 
+def count_records_without_tokenization(project_id: str) -> int:
+    query = f"""
+    SELECT count(r.id) c
+    FROM ({get_records_without_tokenization(project_id, 0, True)}) r
+    """
+    return general.execute(query).first().c
+
+
+def get_attribute_data_with_doc_bins_of_records(
+    project_id: str, attribute_name: str
+) -> List[Any]:
+    query = f"""
+    SELECT rt.id, r."data" ->> '{attribute_name}' "attribute_data", rt.bytes
+    from record r
+    JOIN record_tokenized rt
+    ON  r.id = rt.record_id
+    WHERE r.project_id = '{project_id}'
+    AND rt.project_id = '{project_id}'
```

Comment:
Styleguide issues

```suggestion
    SELECT 
        rt.id, 
        r."data" ->> '{attribute_name}' "attribute_data", 
        rt.bytes
    FROM record r
    INNER JOIN record_tokenized rt
        ON  r.id = rt.record_id
    WHERE r.project_id = '{project_id}'
    AND rt.project_id = '{project_id}'
```

Reply:
also this will only result in values if the record already has something in the record_tokenized table. Is this wanted?